pkg/controllers/management/authprovisioningv2/cluster.go

```diff
@@ -120,6 +122,16 @@ func (h *handler) enqueueCRTBsForRKEClusters(cluster *v1.Cluster) error {
 		for _, crtb := range crtbs {
 			h.clusterRoleTemplateBindingController.Enqueue(crtb.Namespace, crtb.Name)
 		}
+		prtbs, err := h.projectRoleTemplateBindings.List("", labels.Everything())
```

Comment:
Do you think you could rename this function to reflect this added behaviour? The function is called `enqueueCRTBsForRKEClusters` so either something like `enqueueRoleTemplateBindingsForRKEClusters` or even adding a separate function `enqueuePRTBsForRKEClusters`. It's not an exported function so we don't have to worry about external use.

Reply:
good point! I renamed it to `enqueueRoleTemplateBindingsForRKEClusters`